tools/awslint/lib/cfn-resources.ts

```diff
@@ -68,7 +68,7 @@ export function findCfnResources(assembly: reflect.Assembly): CfnResourceSpec[]
     if (!c.system.includesAssembly(cdkAssembly)) {
       return false;
     }
-    const resourceBaseClass = c.system.findFqn(`${cdkAssembly}.Resource`);
+    const resourceBaseClass = c.system.findFqn('@aws-cdk/cdk.CfnResource');
```

Comment:
Why drop `${cdkAssembly}`?

Reply:
That's because it was added by Romain today, and I missed it when doing the merge from master. Will fix.

Reply:
Yeah that's a merge artifact for sure :)